fili-core/src/main/java/com/yahoo/bard/webservice/data/PartialDataHandler.java

```diff
@@ -39,24 +87,17 @@
      * present for a given combination of request metrics and dimensions (pulled from the API request and generated
      * druid query) at the specified granularity.
      *
-     * @param constraint  Constraint containing all the column names the request depends on
-     * @param physicalTables  the tables whose column availabilities are checked
+     * @param availableIntervals  the intervals available in the tables
      * @param requestedIntervals  The intervals that may not be fully satisfied
      * @param granularity  The granularity at which to find missing intervals
      *
      * @return subintervals of the requested intervals with incomplete data
      */
     public SimplifiedIntervalList findMissingTimeGrainIntervals(
```

Comment:
My point was more along the lines of "this method didn't exist before, since this signature is changing, so do we need to expose it". So, from that standpoint, this method here doesn't need to be deprecated.

But if we're OK supporting this contract going forward, and we think there's a high likelihood of it being useful outside this class (by someone who would already depend on this class), then it being `public` is fine by me. Otherwise, I'd lean towards hiding it until it's needed.